Guides/Building_AOSP.txt

```diff
@@ -126,6 +126,12 @@ $ prebuilts/misc/linux-x86/ccache/ccache -M 50G (or however much you want).
 Run this command if you used the automatic method of setup above
 $ ccache -M 50G
 
+Since Q, the way of using ccache has changed:
```

Comment:
To be entirely honest, the logic in the commit message that removed the prebuilt `ccache` binary kind of makes sense: https://android.googlesource.com/platform/build/+/7556703f0dfab14c91c03dab1a3c25f6386268d4

It might be wise to discourage use of this from Android 10 onward but the choice is up to you. See my comments below if you want to keep this.